about.html

```diff
@@ -0,0 +1,161 @@
+            Dropbox.
+          </p>
+          <h6 class="h6">
+            Please contact us through our email for any further questions about
+            Bootcamp 2022! <br /><span>wdbootcamp2022@contact.co</span>
+          </h6>
+        </div>
+      </header>
+
+      <!-- LOGGO SECTION  -->
+      <section id="about-logo">
+        <div class="upper">
+          <h2>Web Development Bootcamp 2022 Logo</h2>
+          <hr />
+        </div>
+        <div class="lower">
+          <p>
+            The logo of WD 2022 Bootcamp was created feb 2021 on a the first
+            assembly meeting Take a look at our recent students sharing and
+            discussing through one of our platforms world-wide.
+          </p>
+          <div class="spanner">
+            <i class="fa-solid fa-laptop-code fa-2xl logo"></i>
+            <h2>WD Bootcamp 2022</h2>
+          </div>
+        </div>
+      </section>
+
+      <!-- FEATURED SECTION  -->
+      <section id="about-featured">
+        <div class="upper">
+          <h2>See the Past WD Graduates</h2>
+          <hr />
+        </div>
+        <div class="lower">
+          <p>
+            Take a look at our recent students sharing and discussing through
+            one of our platforms world-wide.
+          </p>
+          <div class="images">
+            <div class="image">
+              <div class="front">
+                <h2>2021</h2>
+                <p>WD 2021 Bootcamp in our main hall</p>
+              </div>
+              <a
+                ><img
+                  class="reds"
+                  src="images/image-1.jpg"
+                  alt="Students-discussing"
+              /></a>
+            </div>
+            <div class="image">
+              <div class="front">
+                <h2>2022</h2>
+                <p>WD 2022 Bootcamp in our main Ohio</p>
+              </div>
+              <a
+                ><img
+                  class="reds"
+                  src="images/image-2.jpg"
+                  alt="Students-discussing"
+              /></a>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <!-- PARTNERS SECTION  -->
+      <section id="partners">
+        <div class="upper">
+          <h2>Partners</h2>
+          <hr />
+        </div>
+        <div class="lower">
+          <span><i class="fab fa-google"></i>Google</span>
+          <span><i class="fab fa-airbnb"></i>airbnb</span>
+          <span><i class="fab fa-microsoft"></i>Microsoft</span>
+          <span><i class="fab fa-apple"></i>Apple</span>
+          <span><i class="fab fa-free-code-camp"></i>freeCodeCamp</span>
+        </div>
+      </section>
+      <footer>
+        <div class="left">
+          <i class="fa-solid fa-laptop-code fa-2xl"></i>
+          <h3>Bootcamp</h3>
+        </div>
+        <div class="right">
+          <span>2022 Web Dev Malmo. All Rights Reserved</span>
+        </div>
+      </footer>
```

Comment:
- [ ] (Required) It was hard making this a required change because you designed this project nicely. The `footer` for the _**about page**_ should have a `background-color` of `#272a31` and not a white background color. It would be nice to fix this change because your project should look almost identical to the original design, outside the content differences.

- [ ] Kindly remove the **_partner_** section from the _**about page**_ because the template design doesn't have the _**partner section**_. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/157918072-109b452a-5740-4b23-8e38-603e9e9faeac.png)

### Behance Design

![image](https://user-images.githubusercontent.com/77106962/157918270-a6f572fa-e63c-40f8-8b57-45d5eda56418.png)

Reply:
@Gambit142 Thanks for addressing these suggestions, I will make sure to Implement all of them understanding that they will improve my project a lot. Again Thank you for spending the time and stopping by 🙏🏾.

Reply:
You welcome any day anytime @Yazino12 😄